### PR TITLE
Unittesting FTW?

### PR DESCRIPTION
**Pull request by remiolsen**
Decided I wanted to go all out on this package and do unittesting (because, why not? :trollface: )

@ewels Can you check if this travis integration works? Or integration to this repo?

### CODE REVIEW

File: update_kpis/kpiupdater/kpi.py [hunk 1]
```diff
@@ -176,12 +184,12 @@ class ProjectsClosedNWeeks(KPIBase):
...
     """
     def __init__(self):
+        self.dnow = datetime.now()
         self.max_weeks = 4
         self.state = {key: 0 for key in range(0, self.max_weeks)}
 
     def __call__(self, doc):
-        dnow = datetime.now()
-        dweekend = datetime.strptime("{0}-{1}-0".format(*dnow.isocalendar()),"%Y-%W-%w")
+        dweekend = datetime.strptime("{0}-{1}-0".format(*self.dnow.isocalendar()),"%Y-%W-%w")
```
Comment (ewels) [inline update_kpis/kpiupdater/kpi.py:192]: What does the asterisk in `*self` do?

Comment (ewels) [inline update_kpis/kpiupdater/kpi.py:192]: Oh yeah, expands it so that you get both `{0}` and `{1}`, I remember. Nice!

**Comment by ewels** [comment]
Looks awesome! I've just enabled Travis for this repo, so once this merge goes through it _should_ build..

**Comment by remiolsen** [comment]
Thank you :+1:

**Pull request merged**